gulpfile.js

```diff
@@ -104,8 +104,29 @@ var parseTypingsFile = function(folder, file) {
       if (line.indexOf('export') !== -1) {
         line = line.replace('export', '').trim();
       }
+
+      // Remove webdriver types and plugins for now
+      line = removeTypes(line,'webdriver.ActionSequence');
+      line = removeTypes(line,'webdriver.promise.Promise');
+      line = removeTypes(line,'webdriver.util.Condition');
+      line = removeTypes(line,'webdriver.WebDriver');
+      line = removeTypes(line,'webdriver.Locator');
+      line = removeTypes(line,'webdriver.WebElement');
+      line = removeTypes(line,'Plugins');
       contents += line + '\n';
     }
+
   }
   return contents;
 }
+
+var removeTypes = function(line, webdriverType) {
+  var tempLine = line.trim();
+  if (tempLine.startsWith('/**') || tempLine.startsWith('*')) {
```

Comment:
This should be fine because I am trimming the line prior to testing if it starts with a `*`. This should capture the comments in the index.d.ts file.